src/apps/store/services/book.service.ts

```diff
@@ -40,8 +42,16 @@ export type UpdateUserInput = {
 export class BookService {
   constructor(private readonly bookRepository: BookRepository) {}
 
-  async getAll(data: GetBooksInput): Promise<GetBooksOutput> {
-    const result = await this.bookRepository.findAndCount({}, { ...data });
+  async getAll(
```

Comment:
It should be separated into other method instead of adjust the `getAll()` method